test/65285_temperature.js

```diff
@@ -0,0 +1,62 @@
+var chai = require('chai')
+chai.Should()
+chai.use(require('chai-things'))
+chai.use(require('@signalk/signalk-schema').chaiModule)
+var debug = require('debug')('n2k-signalk:test:130312')
+var _ = require('lodash')
+
+var msgs = require('./65285_temperature-2.json')
+
+describe('Temperature: ', function () {
+  it('examples work', function () {
+    var n2kMapper = require('../n2kMapper.js')
+    var full = new (require('@signalk/signalk-schema')).FullSignalK(
+      'urn:mrn:imo:mmsi:230099999'
+    )
+    _.forOwn(msgs, function (msg, key) {
```

Comment:
Your test data file has only one sample, so the loop here does not do much. I would remove it and add something that tests the conversion result’s actual value. As it is the output could have a constant 0 value for example and the test would still be happy.